spec/disk_spec.rb

```diff
@@ -37,7 +37,7 @@
 
       disk = LinuxAdmin::Disk.new :path => '/dev/hda'
       allow(disk).to receive(:run!).and_return(double(:output => fdisk))
-      expect(disk.size).to eq(500.1.gigabytes)
+      expect(disk.size).to eq(500107862016)
```

Comment:
No, that's the issue.  The bytes value is the only thing we can rely on from the fdisk output.
This value would technically be exactly either 465.761741638 GiB (500107862016 / 1024^3) or 500.107862016 GB (500107862016 / 1000^3).

The bug was that we were interpreting the GB as GiB